db-cleanup/airflow-db-cleanup.py

```diff
@@ -1,10 +1,15 @@
 from airflow.models import DAG, DagRun, TaskInstance, Log, XCom, SlaMiss, DagModel, Variable
 from airflow.jobs import BaseJob
 from airflow.models import settings
-from airflow.operators import PythonOperator
+from airflow.operators.python_operator import PythonOperator
```

Comment:
What version of Airflow is this for? Can we support 1.7.1.3 and above? perhaps create another branch as needed?

Reply:
This is working fine 1.7